build/tools.mk

```diff
@@ -12,10 +12,11 @@ TOOL_DIR     ?= tools
 TOOL_CONFIG  ?= $(TOOL_DIR)/tools.go
 
 TOOLS_IMAGE ?= grafana/tempo-ci-tools
+TOOLS_IMAGE_TAG ?= main-d8c69d4
```

Comment:
The `rXX` tags also tag `latest` which is strange, since the `rXX` are always behind `main`.  We can follow up with a better idea later, but this at least will use a tools image which has the expected tools.